variables.tf

```diff
@@ -194,4 +194,9 @@ variable "status_update_queue_message_retention_time_seconds" {
   type        = number
   description = "The number of seconds status_update_queue SQS retains a message in seconds. Maximum value is 14 days."
   default     = 777600 #9 days
+}
+
+variable "api_vpc_endpoint_id" {
```

Comment:
May want to call this just `vpc_endpoint_id` to remove confusion that this is the API Gateway id when in reality it is the NGAP VPC endpoint ID. See previous notes on description updates

Reply:
changed to vpc_endpoint_id